server/webapp/WEB-INF/rails.new/app/views/navigation_elements/_navigation.html.erb

```diff
@@ -1,40 +1,56 @@
 <nav class="top-bar navigation" data-topbar role="navigation">
   <ul class="title-area">
     <li class="name">
-      <a href="#" class="logo"></a>
+      <%= link_to '', pipeline_dashboard_path, class: "logo" -%>
     </li>
-    <!-- Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone -->
-    <li class="toggle-topbar menu-icon"><a href="#"><span>Menu</span></a></li>
   </ul>
 
   <section class="top-bar-section">
-    <ul class="right">
+    <ul class="current-user">
+      <% unless @user.anonymous? %>
         <li class="has-dropdown">
-            <a href="#" class="user">admin</a>
-            <ul class="dropdown">
-                <li><a href="#">Preferences</a></li>
-                <li><a href="#">Sign out</a></li>
-            </ul>
+          <a href="#" class="user"><%= @user.display_name %></a>
+          <ul class="dropdown">
+            <li><%= link_to 'Preferences', url_for_path('/tab/mycruise/user') %></li>
+            <li><%= link_to 'Sign out', url_for_path('auth/logout') %></li>
+          </ul>
         </li>
+      <% end %>
+      <li>
+        <%= link_to("Help", "http://www.go.cd/documentation/user/current", :target => "_blank") %>
+      </li>
     </ul>
-    <ul class="left">
-      <li class="active" ><a href="#">Pipeline</a></li>
-      <li><a href="#">Environments</a></li>
-      <li><a href="#">Agents</a></li>
-      <li class="has-dropdown not-click">
```

Comment:
If I'm not mistaken that 'not-click' is needed for some hovering/clicking issues. Can you check?